app/models/manageiq/providers/amazon/cloud_manager/event_parser.rb

```diff
@@ -1,48 +1,45 @@
 module ManageIQ::Providers::Amazon::CloudManager::EventParser
-  # these have their own EmsEvent classes in automate
-  INSTANCE_EVENTS = %w(
-    AWS_EC2_Instance_CREATE
-    AWS_EC2_Instance_UPDATE
-    AWS_EC2_Instance_DELETE
-    AWS_EC2_Instance_running
-    AWS_EC2_Instance_shutting-down
-    AWS_EC2_Instance_stopped
-  ).to_set.freeze
-
-  def self.map_event_type(aws_event_type)
-    if INSTANCE_EVENTS.include?(aws_event_type)
-      aws_event_type
-    else
-      'ConfigurationItemChangeNotification'
-    end
+  def self.parse_config_event!(event, event_hash)
+    event_hash[:message]                   = event["configurationItemDiff"]
```

Comment:
That being said, we could have something like show me more events. But I think the way of listing them in settings is supposed to make it more understandable when showing it.

But that being said, I would like to see :all in timelines and do a smart grouping of the events. So you don't see 1 event but a (30 events of VmCreated) which will be a list to those events.

With a smart grouping we can display large number of events and have a nice overview of what is happening.